alert-policies/flutter-android/RequestsperMinute.yml

```diff
@@ -0,0 +1,41 @@
+# Name of the alert
+name: Requests per Minute
+
+# Description and details
+description: |+
+  This alert is triggered when the requests per minute is above 60 for at least 1 min.
```

Comment:
@pkudikyala In this description why 60 requests per minute is there any reason or some where mentioned for these type of requests per minute section?

Reply:
I have updated it as per your comments.
Thank you...